charts/ssh/templates/ssh.deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "ssh.fullname" . }}
+  labels:
+    {{- include "ssh.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.ssh.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "ssh.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "ssh.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+          - name: RABBITMQ_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: RABBITMQ_USERNAME
+                name: ssh-env
+          - name: RABBITMQ_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: RABBITMQ_PASSWORD
+                name: ssh-env
+          - name: JWTSECRET
+            valueFrom:
+              secretKeyRef:
+                key: JWTSECRET
+                name: ssh-env
+          - name: SERVICE_NAME
+            value: ssh
+          - name: CRONJOBS
+          - name: API_DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: API_DB_PASSWORD
+                name: ssh-env
+          securityContext: {}
+          image: "{{ .Values.ssh.image.repository }}:{{ .Values.ssh.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.ssh.image.pullPolicy }}
+          ports:
+            - name: ssh
+              containerPort: {{ .Values.ssh.containerPort }}
+              protocol: TCP
+          livenessProbe:
```

Comment:
You probably don't need to set all these values, because the defaults are pretty sensible. e.g. this might be enough:
```
livenessProbe:
  tcpSocket:
    port: ssh
```
Also some of these settings are set to default values. e.g. `failureThreshold` defaults to 3. see [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) for the defaults.

Same for the `readinessProbe`.